src/modules/users/CMakeLists.txt

```
@@ -6,6 +6,16 @@
 find_package(${qtname} ${QT_VERSION} CONFIG REQUIRED Core DBus Network)
 find_package(Crypt REQUIRED)
 
+# Check for crypt_gensalt
+if(Crypt_FOUND)
+    list(APPEND CMAKE_REQUIRED_LIBRARIES crypt)
```

You'll want to undo this after the call to `check_symbol_exists()`, so first, make a copy:
```suggestion
    set(_old_CRL "${CMAKE_REQUIRED_LIBRARIES}")
    list(APPEND CMAKE_REQUIRED_LIBRARIES crypt)
```